storage/src/checker.rs

```diff
@@ -39,6 +39,12 @@ impl<S: WritableStorage> NodeStore<Committed, S> {
         // 2. traverse the trie and check the nodes
         if let Some(root_address) = self.root_address() {
             // the database is not empty, traverse the trie
+            if root_address.get() % 16 != 0 {
```

Comment:
Use AREA_SIZES[0] instead of 16. For completeness, you may want to write a test in the file that defines AREA_SIZES that verifies that all AREA_SIZES are multiples of the first one, something simple like this (untested):
```
assert_eq!(
        AREA_SIZES.first(),
        AREA_SIZES.iter().reduce(|lo, hi| if hi % lo == 0 { lo } else { hi })
    );
```